Makefile

```diff
@@ -157,7 +157,7 @@ FIND_PATHS := $(addprefix -o -path $(LIB_DIR)/,$(IGNORE_CLEANUP_LIBS))
 FIND := find $(PROJECT_DIR) $(LIBRARY_DIR) \
 			  \( $(wordlist 2,$(words $(FIND_PATHS)),$(FIND_PATHS)) \) -prune -o \
 				-iname "*.[ch]" -print
-FIND_MOD_NEW := git diff --name-only --diff-filter=ACMRT -- *.{c,h}
+FIND_MOD_NEW := git diff origin/master --name-only --diff-filter=ACMRT -- '*.c' '*.h'
```

Comment:
Is this the part that checks for anything that's different from the HEAD?

Reply:
Just curious how this works

Reply:
Yea adding `origin/master` should compare to the HEAD of that branch. By default this would only find uncommitted/unstaged files. So this should hopefully be better assuming you've fetched.

The `*.c *.h` change lets git filter only C files. I think before it was trying to shell glob, which doesn't work here